corehq/apps/repeaters/models.py

```diff
@@ -509,85 +552,39 @@ def try_now(self):
         return not self.succeeded
 
     def get_payload(self, save_failure=True):
-        try:
-            return self.repeater.get_payload(self)
-        except ResourceNotFound as e:
-            # this repeater is pointing at a missing document
-            # quarantine it and tell it to stop trying.
-            logging.exception(
-                u'Repeater {} in domain {} references a missing or deleted document!'.format(
-                    self._id, self.domain,
-                ))
-
-            if save_failure:
-                self._payload_exception(e, reraise=False)
-        except Exception as e:
-            if save_failure:
-                self._payload_exception(e, reraise=True)
-            else:
-                raise
+        warnings.warn("RepeatRecord.get_paylod is deprecated. Use Repeater._get_payload instead.")
```

Comment:
Ah you're both right, thanks. It should say "Use Repeater.get_payload_and_handle_exception instead" now.